cnd/src/db/schema.rs

```diff
@@ -9,3 +9,99 @@ table! {
         role -> Text,
     }
 }
+
+table! {
+   rfc003_bitcoin_ethereum_bitcoin_ether_request_messages {
+       id -> Integer,
+       swap_id -> Text,
+       bitcoin_network -> Text,
+       ethereum_chain_id -> Integer,
+       bitcoin_amount -> Text,
+       ether_amount -> Text,
```

Comment:
:question: Could/should amounts be `Integer`s?

Reply:
We are storing wei and satoshis, storing those as integers could be dangerous in regards to overflows.

Reply:
True that.